docs/en/sql-reference/functions/url-functions.md

```diff
@@ -117,6 +117,10 @@ Returns the part of the domain that includes top-level subdomains up to the “f
 
 For example, `cutToFirstSignificantSubdomain('https://news.yandex.com.tr/') = 'yandex.com.tr'`.
 
+### port(URL) {#port}
```

Comment:
Probably `port(URL, default = 0)` is more user-friendly?

Reply:
An SQL-ish way would be to use null + coalesce. We don't have nulls, so maybe we need coalesce but for zeros...
This PR is fine either way IMO.

Reply:
>null + coalesce

Good idea, I make it return `UInt16` over `Nullable(UInt16)` intentionally:
- 0 port for URL is enough indicator that it is empty
- and to avoid extra nullable conversion

>Probably port(URL, default = 0) is more user-friendly?

Ok, thinking about this a little bit more, and seems that this can be useful, i.e. making it 80 for http, but I was thinking about making it const, hence something like will not work anyway `with 'http://foobar.com/' as url select port(url, if(protocol(url) == 'http://', 80, 443))`

Reply:
Added default_port but only as const